packages/cli/src/commands/generate/helpers.js

```diff
@@ -192,3 +197,45 @@ export const forcePluralizeWord = (word) => {
 
   return pluralize.plural(word)
 }
+
+export const validatePlural = (plural, singular) => {
+  const trimmedPlural = plural.trim()
+  if (trimmedPlural === singular) {
+    return 'Plural can not be same as singular.'
+  }
+  if (trimmedPlural.match(/[\n\r\s]+/)) {
+    return 'Only one word please!'
+  }
+  // Control Char u0017 is retured if default input is cleared in the prompt using option+backspace
+  // eslint-disable-next-line no-control-regex
+  if (trimmedPlural.match(/^[\n\r\s\u0017]*$/)) {
+    return 'Plural can not be empty.'
+  }
+  return true
+}
+
+// Ask user for plural version, if singular & plural are same for a word. For example: Pokemon
+export const ensureUniquePlural = async (model) => {
+  if (!isWordNonPluralizable(model)) {
+    return
+  }
+
+  const initialPlural = model.slice(-1) === 's' ? `${model}es` : `${model}s` // News => Newses; Post => Posts
+  const promptResult = await prompts({
+    type: 'text',
+    name: 'plural',
+    message: `Cannot determine the plural of "${model}". \nTo continue, the generator requires a unique plural form:`,
```

Comment:
@callingmedic911 This is looking great and is so very close! Just one final thing I noticed -- when running the `destroy` command, the message above is out of context and no longer makes complete sense. (see screenshot below) I suggest that when used for `destroy`, something like the following message should display instead:
```js
message: `Cannot determine the plural of "${model}" originally used to generate scaffolding. \nTo continue, the destroy command requires the plural form:`,
```

<img width="612" alt="Screen Shot 2021-06-21 at 10 16 43 AM" src="https://user-images.githubusercontent.com/2951/122802856-bdf93100-d27a-11eb-81ee-32c8ec5973c7.png">

Reply:
Oh right! The part in the message "originally used to generate scaffolding" feels very specific to `scaffold`. Do you think it would be okay to display this for `destory sdl` or `service` too?

Thanks a lot for the help @thedavidprice! 🙂

Reply:
Updated the message for destroyer prompt like you suggested. But do let me know your thoughts on above. :)